lib/requestIdleCallbackPolyfill.js

```diff
@@ -0,0 +1,24 @@
+// A poyfill for requestIdleCallback.
+//
+// For: IE 11, Edge, and Firefox
```

Comment:
Note according to [caniuse](https://caniuse.com/#feat=requestidlecallback) Firefox supports this but Safari does not.

Reply:
Oh, nice. I must have glanced too quickly at the table on Mozilla.